bob-common/src/mapper.rs

```diff
@@ -121,6 +121,47 @@ impl Virtual {
         self.vdisks.get(&id).expect("vdisk not found").nodes()
     }
 
+    fn find_support_node_index(
```

Comment:
I don't like this name. Maybe `try_find_support_node_offset_at_one_pass`?

Reply:
Done.